### DISCUSSION TITLE
Custom sirens

### DISCUSSION THREAD

**Discussion by TLW2019** [matsn0w/IIS-EmergencyLS-ELS-FiveM#146] [Q&A] [answered]
Would server side sound and sirens work in the place of wm sirens i ask because SAS uses diffrent audio banks and has more audio banks to accompany multiple sirens for diffrent departments

**Comment by matsn0w** [comment]
Can you please make your question a little clearer?

**Reply by TLW2019** [reply]
Of course so are we locked to using wm sirens or are we able to use server side sounds and sirens in its place

**Reply by matsn0w** [reply, marked as answer]
Hi, sorry for my late reply.

MISS ELS is designed for WMServerSirens, but it works with any other server sided sirens as well (although I haven't tested...).

Specify the name of the audio bank in your `config.lua` file (section `Config.AudioBanks`) and you're good to go! Also don't forget to set the correct audio names for each light stage.

In the [VCF Configurator](https://matsn0w.github.io/MISS-ELS), make sure to tick the 'Use WMServerSirens' box, you can then specify the name of the audio string and the soundset. Refer to the docs of your siren pack for more info.

Good luck! Let me know if it worked. Which pack are you planning to use?

**Reply by TLW2019** [reply]
custom pack i have it works with all fivem servers already and i have the names i need for it also am im correct in saying this works with LVC or is it a replacement?

**Reply by matsn0w** [reply]
MISS-ELS is not compatible with LVC or any other ELS resource.
